packages/validator/src/commands/utils.ts

```diff
@@ -5,13 +5,27 @@ import { Log } from '../types';
 import { unknownConversion } from '../utils/messages';
 
 const validatorTemplatesPath = path.resolve(__dirname, '../../templates');
-const conversionsPath = path.resolve(__dirname, '../../templates/conversion');
+const conversionsPath = path.resolve(__dirname, '../../conversions');
 
 export const templateVersions = fs
   .readdirSync(validatorTemplatesPath, { withFileTypes: true })
-  .filter((dirent) => dirent.isDirectory() && dirent.name !== 'conversion')
+  .filter((dirent) => dirent.isDirectory())
   .map((dirent) => dirent.name)
-  .sort((a: any, b: string) => b.localeCompare(a, undefined, { numeric: true }));
+  .sort((a: string, b: string) => {
+    if (!a.match(/^[0-9\.]+$/)) {
+      if (!b.match(/^[0-9\.]+$/)) {
+        return 0;
```

Comment:
But this means that each version that is not in a semantic versioning format (something like just text, e.g. `alpha`) is evaluated as the same. I think we should sort those alphabetically as well.

Reply:
Yes this is intentional, as it was discussed versioning will be primarily numerical from now on, validator will be focused on the numerical versions so the latest version will be `0.1.0` until the next numerical version will come out. It will just throw all the alphabetical versions at the very end, so that they are not considered, sorting them alphabetically does not make much sense as for example `beta` would be ranked higher than `pre-alpha`, to correctly sort versions we would have to import a library, which I believe is a bit overkill and focusing only on numerical versions should be sufficient.

Reply:
I understand and I agree that we will focus on the semantic versioning from now on. However, it makes sense to provide a deterministic ordering of all versions (even though migth be wrong as you mentioned). Listing a directory is one of those things that can be very much OS dependent so the order may not be guaranteed. Forcing the ordering in such a case is just overall less error prone.